stylized login page

doddsip opened the pull request:
the following has been stylized:
-form box
-username label
-username input
-password label
-login button
-error message (li)

for the html file:
-created a "div class form-region" login box
-created "username-label" span for the username label styling -created "password-label" span for the password label styling -created "error-text" div for the error text
-gave the username label & password label a <br> tag to put the input form underneath the text -gave the username & password form an extra <br> tag to space the two inputs

doddsip committed:
stylized login page

the following has been stylized:
-form box
-username label
-username input
-password label
-login button
-error message (li)

for the html file:
-created a "div class form-region" login box
-created "username-label" span for the username label styling
-created "password-label" span for the password label styling
-created "error-text" div for the error text
-gave the username label & password label a <br> tag to put the input form underneath the text
-gave the username & password form an extra <br> tag to space the two inputs

TheodoreBagley commented:
Looks good to me and it works properly. One note (and others may disagree) I personally feel like the welcome text should be within the sign box since right now the box shadow makes it look a little weird. everything else looks fantastic!

doddsip committed:
small styling changes + fixes

-moved index.css code regarding the nav to base.css (this changes nothing functionality wise)
-added inventory.css
-added report.css
-both were made so that the underline can appear on the page the user goes to
-removed "element=current" under nav.html for inventory since it's redundant
-implemented the underline of inventory.css in table.html
-implemented the underline of reports.css in reports.html

TheQantas commented:
Lgtm. I agree with Teddy about moving the welcome text.

We should also open a new issue to fix the login page on phones.
![image](https://github.com/user-attachments/assets/be19da2e-ea05-42d3-bc8c-47c827cedc45)

TheodoreBagley committed:
Merge branch 'main' into matthew-f-release
